src/lib.js

```diff
@@ -165,6 +165,7 @@ export default class Migrator {
         this.log('There are no migrations to run'.yellow);
         this.log(`Current Migrations' Statuses: `);
         await this.list();
+        return []
```

Comment:
I think you want this in the `else` block @river-jade 
return [] if it's NOT CLI